ticketutil/bugzilla.py

```diff
@@ -138,10 +180,7 @@ def _create_ticket_request(self, params):
         """
         # Attempt to create ticket.
         try:
-            if self.token:
```

Comment:
Do you not need to add {'token': self.token} to each request call anymore for HTTP Basic?